docs/site/content/docs/edge/assets/prereq-unmanaged-windows.md

```diff
@@ -1,10 +1,9 @@
-| Architecture   | CPU | RAM  | Required software |
-|:---------------|:----|:-----|:------------------|
-| x86_64 / amd64 | 1   | 2 GB | [Docker Desktop](https://www.docker.com/get-started), (Optional) [minikube](https://minikube.sigs.k8s.io/docs/start/) |
+| CPU | RAM  | Required software |
+|:----|:-----|:------------------|
+| 1   | 2 GB | [Docker Desktop](https://www.docker.com/get-started), (Optional) [minikube](https://minikube.sigs.k8s.io/docs/start/) |
```

Comment:
It might be good to keep that Architecture column to keep it explicit that only `x86_64/amd64` architectures are supported.

Reply:
I removed it because the Windows instructions for Managed Clusters don't have it. 

But I can add it back.

Reply:
Yea, there might be (going to bring this up in a second) that we support arm64 and we should make it explicit

Reply:
Do you want me to do the opposite and also add the architecture column to the Managed Clusters page?

Reply:
Let's just keep it here for now. With managed clusters it gets a little trickier because we need to distinguish deploying-from and deploying-to. With unmanaged clusters it's a little more straight forward, so I think having the column here makes it a little more clear.

My 2 cents at least.

Reply:
> It might be good to keep that Architecture column to keep it explicit that only x86_64/amd64 architectures are supported.

I agree - we should keep arch explicit here since it is supported on arm for M1 macs